plugins/admission/pkg/admission/handler.go

```diff
@@ -15,10 +15,55 @@ import (
 	"github.com/fairwindsops/insights-plugins/plugins/admission/pkg/polaris"
 )
 
+// webhookFailurePolicy stores the states of a Kubernetes
+// ValidatingWebhookConfiguration webhook failurePolicy.
+type webhookFailurePolicy int
+
+// webhookFailurePolicy* constants represent possible values for a Kubernetes
+// ValidatingWebhookConfiguration webhook failurePolicy.
+const (
+	webhookFailurePolicyIgnore = iota
+	webhookFailurePolicyFail   = iota
```

Comment:
I think you're supposed to omit the second iota...

https://golangbyexample.com/iota-in-golang/